src/quiz/use-quiz.ts

```diff
@@ -17,10 +18,12 @@ export const useQuiz = <AnswerT extends number | string>({
 	validationMessages,
 	onSuccess,
 	onFailure,
+	passingGrade = 100,
 }: Props<AnswerT>) => {
 	const [questions, setQuestions] =
 		useState<Question<AnswerT>[]>(initialQuestions);
 	const [correctAnswerCount, setCorrectAnswerCount] = useState(0);
+	const [grade, setGrade] = useState(0);
```

Comment:
Good question! I think `null` (or perhaps `undefined`) makes sense, mostly because it doesn't seem right to say that someone has a grade or number of correct answers until they have been graded.

One thing to consider is a return type like this:

```ts
{ validated: true, grade: number, correctAnswerCount: number, ... } | { validated: false, grade: never, correctAnswerCount: never,  ... }
```

That way the client can, in principle, just check the value of `validated`, rather than having to account for both `grade` and `correctAnswerCount` being nullable. I say "in principle", because I'm not 100% sure how the code would look in practice, but I think it should work.